packages/gatsby/src/bootstrap/index.js

```diff
@@ -175,10 +175,12 @@ module.exports = async (args: BootstrapArgs) => {
       data
     `)
   }
-
+  const cacheDirectory = `cacheDirectory`
```

Comment:
I think this should be 
```suggestion
  const cacheDirectory = `${program.directory}/.cache`
```
?

Reply:
Uhhh... Yeah 😓